presto-docs/src/main/sphinx/optimizer/cost-based-optimizations.rst

```diff
@@ -69,6 +69,19 @@ The valid values are:
  * ``BROADCAST`` - broadcast join distribution is used for all joins
  * ``PARTITIONED`` - partitioned join distribution is used for all join
 
+-----------------------------
```

Comment:
remove `(cherry picked from commit ...)` from the commit message; it's not useful